api/src/main/java/org/openmrs/module/openconceptlab/Utils.java

```diff
@@ -177,4 +181,31 @@ public static UUID version5Uuid(String name) {
 		return new UUID(msb, lsb);
 	}
 
+	public static void setAllowDecimal(ConceptNumeric numeric, Boolean allowDecimal) {
+		try {
+			Method setPrecise = numeric.getClass().getDeclaredMethod("setPrecise", Boolean.class);
+			setPrecise.invoke(numeric, allowDecimal);
+		} catch (NoSuchMethodException | IllegalAccessException | InvocationTargetException e) {
+			try {
+				Method setAllowDecimal = numeric.getClass().getDeclaredMethod("setAllowDecimal", Boolean.class);
+				setAllowDecimal.invoke(numeric, allowDecimal);
+			} catch (NoSuchMethodException | IllegalAccessException | InvocationTargetException e1) {
+				throw new ImportException(e1);
+			}
+		}
+	}
+
+	public static Boolean getAllowDecimal(ConceptNumeric numeric) {
+		try {
+			Method getPrecise = numeric.getClass().getDeclaredMethod("getPrecise");
+			return (Boolean) getPrecise.invoke(numeric);
+		} catch (NoSuchMethodException | IllegalAccessException | InvocationTargetException e) {
+			try {
+				Method getAllowDecimal = numeric.getClass().getDeclaredMethod("getAllowDecimal");
+				return (Boolean) getAllowDecimal.invoke(numeric);
+			} catch (NoSuchMethodException | IllegalAccessException | InvocationTargetException e1) {
+				throw new ImportException(e1);
+			}
+		}
+	}
 }
```

Comment:
I need a "getAllowDecimal" method for some unit tests to pass in both pre-2x and post-2x, so I moved these two methods here so they would be together for easier reference and reuse.